tests/conftest.py

```diff
@@ -36,6 +37,34 @@
 
 assert "airflow" not in sys.modules, "No airflow module can be imported before these lines"
 
+# Clear all Environment Variables that might have side effect,
+# For example, defined in /files/airflow-breeze-config/variables.env
+_AIRFLOW_CONFIG_PATTERN = re.compile(r"^AIRFLOW__(.+)__(.+)$")
+_KEEP_CONFIGS_SETTINGS: dict[str, dict[str, set[str]]] = {
+    # Keep always these configurations
+    "always": {
+        "database": {"sql_alchemy_conn"},
+        "core": {"sql_alchemy_conn"},
+        "celery": {"result_backend", "broker_url"},
```

Comment:
I'm sure that we also need to check why unit tests expects to have this parameters. If it required then I guess we need to move this tests into the integration, if only for some test configuration, then better configure in specific cases. But this just for thinking and maybe resolve as follow up